plugins/builds/index.js

```diff
@@ -65,6 +65,37 @@ function getExternalEvent(currentBuild, pipelineId, eventFactory) {
     return eventFactory.get(eventId);
 }
 
+/**
+ * Fetch builds with given params
+ * @method triggerFetchBuild
+ * @param  {Object}   config                    Parameters of build to fetch
+ * @param  {Factory}  config.buildFactory       Build Factory
+ * @param  {Number}   config.jobID              Job ID
+ * @param  {Number}   config.eventID            Event ID
+ * @return {Promise}
+ * */
+async function triggerFetchBuild(config) {
```

Comment:
why is this function needed ? it's just wrapping a single line, make this inline where it's used